app/dashboard/templates/bounty/details.html

```diff
@@ -444,9 +444,70 @@ <h5 class="bounty-heading">{% trans "Funder" %}</h5>
   <script src="{% static "v2/js/bootstrap.min.js" %}" crossorigin="anonymous"></script>
   <script src="{% static "v2/js/clipboard.js" %}"></script>
   <script>
-    var bootstrapTooltip = $.fn.tooltip.noConflict()
-    var bootstrapModal = $.fn.modal.noConflict()
-    $.fn.bootstrapModal = bootstrapModal
+    let bootstrapTooltip = $.fn.tooltip.noConflict()
+    let bootstrapModal = $.fn.modal.noConflict()
+  </script>
+  <script>
+    $.fn.runTooltip = bootstrapTooltip
+
+    $('[data-toggle="popover"]').popover()
```

Comment:
you are calling popover twice in your html element one by data-toggle and the other via js call.
In the html remove data-toggle and will be call just one time

Reply:
This isn't the issue. As the popover is still loading twice

Reply:
https://github.com/twbs/bootstrap/issues/12563

Reply:
I will check but we are using the last version.